VMBackup/main/Utils/WAAgentUtil.py

```diff
@@ -53,17 +53,27 @@ def searchWAAgentOld():
 pathUsed = 1 
 try:
     agentPath = searchWAAgent()
-    if(agentPath):
-        waagent = imp.load_source('waagent', agentPath)
+    if agentPath is None:
+       pathUsed = 0
+       # Search for the old agent path if the new one is not found
+       agentPath = searchWAAgentOld()
+    if agentPath:
+        try:
+            # For Python 3.5 and later, use importlib
+            import importlib.util
```

Comment:
will it make sense to have specific files that have legacy version of methods that apply to specific python version and main code calls those files-functions as required?

Reply:
Discussed offline.